### ISSUE TITLE
Signup: Domains/URL step does not load in IE

### ISSUE THREAD

**Issue by eurello** [Automattic/wp-calypso#1756]
In Internet Explorer 11 in Windows 7 and Windows 10, the https://wordpress.com/start/domains step to "Add New WordPress" does not load, it just spins at the greyed-out suggestions: 

![ie](https://cloud.githubusercontent.com/assets/8698879/11882693/36c042f6-a4cb-11e5-80b2-751914b8ddc4.PNG)

Reported in the WordPress.com forums: 
https://en.forums.wordpress.com/topic/setting-up-blog-2?replies=9

**Comment by designsimply** [comment]
Tested and confirmed, and I see the following console error in IE11 on Win10:

```
SCRIPT5007: Unable to get property '_currentElement' of undefined or null reference
vendor.8f9d44b7ee5d4e6a3e1d.min.js (2,22385)
```

![screen shot 2015-12-17 at thu dec 17 5 56 34 pm](https://cloud.githubusercontent.com/assets/1119271/11884413/b2c09aba-a4e7-11e5-9036-e7fd935ba3f0.png)

Video: https://cloudup.com/c9Anoy5lnvT

**Comment by drewblaisdell** [comment]
This was fixed with #1777
